packages/cli/tsconfig.json

```
@@ -5,7 +5,8 @@
     "rootDir": "src",
     "types": ["ts-expose-internals-conditionally", "node"],
     "outDir": "./dist",
-    "sourceMap": true
+    "sourceMap": true,
+    "allowSyntheticDefaultImports": true
```

This is implied by `--module nodenext`

```suggestion
    "sourceMap": true
```

---

Strange, I remember getting an error or warning without that option but now it seems fine. I removed it in b8d7fa0.